apps.yml

```diff
@@ -1,3 +1,27 @@
+# NOTE: authorized_groups and authorized_users control two things independently:
+#
+# 1. Does the SSO Dashboard display a tile to the user for this application?
+#
+#        No, if display: false.
+#            No SSO users can see the tile.
+#
+#        Maybe, if display: true.
+#            If groups/users are defined,
+#                The user must be a member of them to see the tile.
+#            Otherwise,
+#                All SSO users (including the general public) can see the tile.
+#
+# 2. Does Auth0 restrict SSO access to this application by group or user?
+#
+#        No, if client_id is missing.
+#            All SSO users (including the general public) can access the app.
+#
+#        Maybe, if client_id is defined.
+#            If groups/users are defined,
+#                The user must be a member of them to access the application.
+#            Otherwise,
+#                All SSO users (including the general public) can access the app.
```

Comment:
All SSO users (including the general public) have [passed the first stage of access control](https://github.com/mozilla-iam/mozilla-iam/#2-stage-access-validation) and the only thing that governs user access is the second stage access control performed by the relying party.

Reply:
"can access the app" isn't the best abbreviation for that. I'm not sure what is — perhaps "can continue to the app", so that we're not implying statements about app access?